### PR TITLE
Update client/playbook_run.go

### PR DESCRIPTION
**Pull request by lkyuchukov**
#### Summary
Update `client/playbook_run.go` to match `server/playbook_run.go`

#### Ticket Link
Fixes: https://github.com/mattermost/mattermost-server/issues/19994

#### Checklist

- ~~[] Telemetry updated~~
- ~~[ ] Gated by experimental feature flag~~
- ~~[ ] Unit tests updated~~

### CODE REVIEW

**Comment by lkyuchukov** [comment]
@gbochora can you please review this PR ?

**Comment by gbochora** [comment]
@lkyuchukov, thanks for your contribution. Sorry for the delay in reply. I plan to review this PR tomorrow.

File: client/playbook_run.go [hunk 1]
```diff
@@ -10,6 +13,7 @@ const Me = "me"
 type PlaybookRun struct {
 	ID                                   string          `json:"id"`
 	Name                                 string          `json:"name"`
+	Summary                              string          `json:"summary"`
 	Description                          string          `json:"description"`
```
Comment (gbochora) [inline client/playbook_run.go:17]: I think we should remove the `Description` field, it is no longer in the `app.PlaybookRun` structure.

File: client/playbook_run.go [hunk 2]
```diff
@@ -24,8 +28,12 @@ type PlaybookRun struct {
 	PlaybookID                           string          `json:"playbook_id"`
 	Checklists                           []Checklist     `json:"checklists"`
 	StatusPosts                          []StatusPost    `json:"status_posts"`
+	CurrentStatus                        string          `json:"current_status"`
+	LastStatusUpdateAt                   int64           `json:"last_status_update_at"`
 	ReminderPostID                       string          `json:"reminder_post_id"`
 	PreviousReminder                     time.Duration   `json:"previous_reminder"`
+	ReminderTimerDefaultSeconds          int64           `json:"reminder_timer_default_seconds"`
+	StatusUpdateEnabled                  bool            `json:"status_update_enabled"`
 	BroadcastChannelID                   string          `json:"broadcast_channel_id"`
```
Comment (gbochora) [inline client/playbook_run.go:37]: The `BroadcastChannelID` field is also no longer in the `app.PlaybookRun structure`

File: client/playbook_run.go [hunk 3]
```diff
@@ -36,6 +44,16 @@ type PlaybookRun struct {
 	InvitedGroupIDs                      []string        `json:"invited_group_ids"`
 	TimelineEvents                       []TimelineEvent `json:"timeline_events"`
 	CategorizeChannelEnabled             bool            `json:"categorize_channel_enabled"`
```
Comment (gbochora) [inline client/playbook_run.go:46]: The `CategorizeChannelEnabled` field is also no longer in the app.PlaybookRun structure

File: client/playbook_run.go [hunk 4]
```diff
@@ -36,6 +44,16 @@ type PlaybookRun struct {
...
+	Retrospective                        string          `json:"retrospective"`
+	RetrospectivePublishedAt             int64           `json:"retrospective_published_at"`
+	RetrospectiveWasCanceled             bool            `json:"retrospective_was_canceled"`
+	RetrospectiveReminderIntervalSeconds int64           `json:"retrospective_reminder_interval_seconds"`
+	RetrospectiveEnabled                 bool            `json:"retrospective_enabled"`
+	MessageOnJoin                        string          `json:"message_on_join"`
+	ParticipantIDs                       []string        `json:"participant_ids"`
+	CategoryName                         string          `json:"category_name"`
+	MetricsData                          []RunMetricData `json:"metrics_data"`
 }
```
Comment (gbochora) [inline client/playbook_run.go:57]: I think the `app.PlaybookRun.DefaultOwnerID` field is missing here.

File: client/playbook_run.go [hunk 5]
```diff
@@ -222,6 +240,12 @@ type GetPlaybookRunsResults struct {
```
Comment (gbochora) [inline client/playbook_run.go:238]: `Disabled` field is no longer in the `app.GetPlaybookRunsResults`

**Review by gbochora — APPROVED** [review]
Nice work, thanks @lkyuchukov!

**Review by agarciamontoro — APPROVED** [review]
This looks good to me, thank you for your work :tada:

**Pull request merged**